workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -79,3 +80,31 @@ If the `delegateIp` option is used, then - if available - an IP address assigned
 |Options||
 |--|--|
 | `delegateIp` | `pod` backend only: The IP assigned to the pod will be delegated to the VM using DHCP |
+
+### `slirp` connection
+
+This connection will use the qemu user interface.
+Let the VM be exposed like a process which would be running inside a container. The requirement is that a process will bind to a port.
+
+It is a known issue that SLIRP is not recommended for anything in production.
+
+All the egress connection will came from the qemu process and all the ingress connection will go to qemu process.
+
+|Feature||
+|--|--|
+| Supported protocols | TCP/UDP only |
+
+|Options||
+|--|--|
+| `ports` | Expose ports  |
+
+#### Ports
```

Comment:
Instead of Ports ake this an example seciton:

Please add an example, then keep the table below to explain the example

Reply:
Done